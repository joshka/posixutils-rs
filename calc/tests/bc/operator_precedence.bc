a = 3.5
b = 9.78
c = 4

++c * d
-a ^ 2
a * b + c
b - a / c
a * b / c + a
c ^ 2 * a
(a + b) * c
((a + b) * c - a) / b
2 + a += b

if (a + b == b + a) {
	"if works"
}

quit
