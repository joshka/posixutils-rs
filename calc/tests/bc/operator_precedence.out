0
12.2
39.23
9.78
9.5
87.5
66.40
6
15.28
if works